Contribution docs reorganization (#81)
Paperback-iOS/website: src/help/contribution/README.md

---
title: Contribute to Paperback
lang: en-US
---

:::guide Contribute to Paperback

> Welcome to Paperback contributing guide. If Paperback initially is a closed source application, it involves a pretty important open source ecosystem. Everyone is welcome to contribute and help in our various project!

Reviewer: ```suggestion
Welcome to Paperback contribution guide. While the Paperback iOS application is a closed source application, it involves a pretty important open source ecosystem. Everyone is welcome to contribute and help in our various project!
```

Reviewer 2: ```suggestion
Welcome to the Paperback contribution guide. While the Paperback app is closed source, the open source ecosystem forms a crucial part of the app. Everyone is welcome to contribute and help on the various projects.
```

I feel this is better.

Author: I would personally prefer Vibrant's "it involves a pretty important open source ecosystem." as the ecosystem is not restricted to the application. There is indeed a website, a backup converter a future paperback-web that are open source projects but not part of the app nor needed and used by it.

For "help in our various project!" or "help on the various projects." I honestly don't know. Using "our" is a bit more personal: our teams are invested in the project, while "the" has a nice neutral tone. A bit more reflection is needed. What would you think?

:::

# Paperback ecosystem

> You will find all our projects in our [GitHub organization](https://github.com/Paperback-iOS). Discussions usually take place in the development channels of our Discord server, you may be interested in joining it.

Reviewer: ```suggestion
You will find all our projects in our [GitHub organization](https://github.com/Paperback-iOS). Additionally, discussions usually take place in the development channels of our Discord server. You're more than welcome to join in on the conversation!
```

Reviewer 2: ```suggestion
You will find all of our projects in our [GitHub organization](https://github.com/Paperback-iOS). Additionally, discussions usually take place in the development channels of our Discord server. You're more than welcome to join in on the conversation!
```

---

Here are some of our main projects:

 * **Paperback extensions**  

>   Extensions for the application are developed and maintained by the community.  

Reviewer: ```suggestion
  Extensions for the application are developed and maintained by the community. These currently consist of sources for pulling content into the app, but in the near future, this will also include manga tracker implementations.
```

  Get started with our [source development guide](extension-development/).

 * **Paperback website**  
  Any help improving this [website](https://github.com/Paperback-iOS/website/) is definitely welcome.

 * **Paperback extensions-common**  

>    Common models and type definitions for developing a Paperback source.

Reviewer 2: ```suggestion
   Contains the models and type definitions necessary for developing a Paperback extension.
```
